pkg/client/preflight.go

```diff
@@ -33,22 +33,10 @@ import (
 	"k8s.io/client-go/kubernetes"
 )
 
-const (
```

Comment:
Similar question here, what happened to the `const` block, is this unused code or is it moved somewhere else ?

Reply:
iirc, this one was never really used for anything, so maybe it was part of scaffolding which didn't end up being used.